content/blog/2016-10-12-interview-with-justwatch.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Interview with JustWatch
+created_at: 2016-10-12
+kind: article
+author_name: Brian Brazil
+---
+
+*Continuing our series of interviews with users of Prometheus, JustWatch talks
+about how they established their monitoring.*
+
+## Can you tell us about yourself and what JustWatch does?
+
+For consumers, [JustWatch](https://www.justwatch.com) is a streaming search
+engine that helps to find out where to watch movies and TV shows legally online
+and in theaters. You can search movie content across all major streaming
+providers like Netflix, HBO, Amazon Video, iTunes, Google Play, and many others
+in 17 countries.
+
+For our clients like movie studios or Video on Demand providers, we are an
+international movie marketing company that collects anonymized data about
+purchase behavior and movie taste of fans worldwide from our consumer apps. We
+help studios to advertise their content to the right audience and make digital
+video advertising a lot more efficient in minimizing waste coverage.
+
+![JustWatch logo](/assets/blog/2016-10-12/JW_logo_long_black.jpg)
+
+Since our launch in 2014 we went from zero to one of the largest 20k websites
+internationally without spending a single dollar on marketing - becoming the
```

Comment:
Typography nit: The dashes (" - " surrounded by blanks) should be an n-dash or even m-dash ("–" or "—"), depending on your typographical style. There are quite a few in this text.